domains/primitives/runtime/src/lib.rs

```diff
@@ -70,6 +70,17 @@ where
     }
 }
 
+#[cfg(feature = "std")]
```

Comment:
This doesn't have to be defined in runtime primitives as it's not really shared across the domain runtimes. Just make it a helper function in cli.

Reply:
it is shared across core domain cli and system domain cli. Would prefer keeping it here as is and only making it available in std

Reply:
Shared across the cli is not equivalent to being shared between the runtime code, I don't see why it has to be defined here.